package/upgrade/upgrade_manifests.sh

```diff
@@ -1257,6 +1257,79 @@ sync_containerd_registry_to_rancher() {
   kubectl annotate --overwrite=true setting.harvesterhci.io containerd-registry "harvesterhci.io/upgrade-patched=$REPO_HARVESTER_VERSION"
 }
 
+upgrade_rbac() {
+
+  # only versions before v1.3.0 that upgrading to v1.3.0 need this patch
+  if [[ ! "${UPGRADE_PREVIOUS_VERSION%%-rc*}" < "v1.3.0" ]]; then
+    echo "Only versions before v1.3.0 need this patch."
+    return
+  fi
+
+  full_name=$(kubectl get rolebinding -A -o wide |grep "harvesterhci.io:cloudprovider" |awk '{print $5}')
```

Comment:
Is it possible to make the search more reliable, like using the correct namespace/name or labels?

Reply:
I think that because the guest cluster can configure the namespace on the creation, it seems we do not have a specific way to filter out what we need.

Reply:
Or use yq to filter the output? e.g.:
```
kubectl get rolebinding -o yaml | yq '.items[] | select (.roleRef.name == "harvesterhci.io:cloudprovider" and .roleRef.kind == "ClusterRole") | .metadata.name'
```

Reply:
```suggestion
  full_names=$(kubectl get rolebinding -A -o wide |grep "harvesterhci.io:cloudprovider" |awk '{print $5}')
```
suggest change to names

Reply:
I add a new command to get the full_names.
Please help to recheck, thanks!